Crackmes/Write-ups/OMTG-Android-App/writeup_Diving_into_mobile_cryptography_using_dynamic_instrumentation_with_frida.md

```diff
@@ -143,6 +149,7 @@ With the terminal pointing to the directory where this files is we will run the
 `frida -U sg.vp.owasp_mobile.OMTG_Android -l hook_decryptString.js`
 
 The Frida CLI will open, connect to the device (`-U`) and will load the script (`-l`).
+The Frida CLI will open, connect to the test device over USB (`-U`), target a specific application under test (`sg.vp.owasp_mobile.OMTG_Android`) & load load the hooking script (`-l`).
```

Comment:
* The added line is partially the same as the previous line. Maybe the previous line should be removed?
* Please use "and" instead of &.
* This has "load load" twice.